backend/src/controllers/action-items.controller.ts

```diff
@@ -25,6 +25,20 @@ class ActionItemsController extends BaseController<IWeeklyActionItems> {
 
     return res.status(httpStatus.OK).send(weeklyActionItems);
   }
+
+  async updateActionItem(req: AuthRequest<IActionItem>, res: Response<void>) {
+    const { id } = req.query;
+    const actionItem: IActionItem = req.body;
+
+    const result = await WeeklyActionItemsModel.updateOne(
```

Comment:
Add businessId to query to make sure only the action item owner can mark as complete